### PR TITLE
Include scaleFUN and suppress warnings in setup_graphics.R

### PR DESCRIPTION
**Pull request by SaranjeetKaur**
The `setup_graphics.R` file helps set up a new plotting device and create a customised theme for ggplot2.

### CODE REVIEW

**Comment by SaranjeetKaur** [comment]
pre_commit is not yet setup for this project in R. So please ignore the failed pre_commit checks for now.

File: man/suppressWarnings.Rd
Comment (jamesturner246) [inline man/suppressWarnings.Rd:1]: Is there a real reason to suppress warnings? typically we'd want building to explicitly fail, even on warnings.

Comment (SaranjeetKaur) [reply man/suppressWarnings.Rd:1]: I agree with you @jamesturner246 - ideally, I would avoid warning suppression.

As of now, I have only reused the code which the researchers shared with us. There isn't much code written yet to test out why they needed the warning suppression

Comment (jamesturner246) [reply man/suppressWarnings.Rd:1]: Ah, I see! That doesn't sound promising. Okay, go ahead. :+1:

**Review by jamesturner246 — APPROVED** [review]
Long as we can eventually plan to tidy up the code and droip the suppress warning dependency, I'm fine with this. Go ahead. :+1:

**Review by alexdewar — COMMENTED** [review]
So far as I understand it, LGTM!

**Comment by SaranjeetKaur** [comment]
Thank you both! 
As suggested by @jamesturner246, I have commented the `suppressWarnings()` in the `setup-graphics.R` file and made an issue https://github.com/imperialCHEPI/healthgpsrvis/issues/13 to check the reason for why it was needed in the first place.

**Comment by SaranjeetKaur** [comment]
I am merging this PR for now!

**Pull request merged**